src/util-cpu.c

```diff
@@ -77,7 +78,16 @@ uint16_t UtilCpuGetNumProcessorsConfigured(void)
 #elif OS_WIN32
 	long nprocs = -1;
 	const char* envvar = getenv("NUMBER_OF_PROCESSORS");
-	nprocs = (NULL != envvar) ? atoi(envvar) : 0;
+    if (envvar != NULL) {
+        if (ByteExtractStringInt64(&nprocs, 10, 0, (const char *)envvar) < 0) {
```

Comment:
Cast on `envar` not needed.

However, the first arg is expected to be `i64_t`, not `long`.